pkg/util/selinux/selinux.go

```diff
@@ -16,12 +16,14 @@ limitations under the License.
 
 package selinux
 
-// chconRunner knows how to chcon a directory.
-type ChconRunner interface {
+// SelinuxContextRunner knows how to chcon of a directory and
+// how to get the selinux context of a file.
+type SelinuxContextRunner interface {
```

Comment:
@pmorie I changed because I added another function which gets the selinux context of a file, so it's not only chcon.